pkgs/by-name/wt/wtf/package.nix

```
@@ -0,0 +1,34 @@
+{ pkgs ? import <nixpkgs> { }, lib ? pkgs.lib, fetchFromGitHub, }:
```

Are you building the package using `nix-build -A wtf`? This 2 defaulted values should not be required

```suggestion
{ stdenv, lib, fetchFromGitHub, }:
```